packages/video-intelligence/src/index.js

```diff
@@ -27,10 +21,12 @@ var gapic = {
   v1beta2: require('./v1beta2'),
 };
 var gaxGrpc = require('google-gax').grpc();
+var googleProtoFiles = require('google-proto-files');
```

Comment:
I don't think this is needed in this file? Was this generated?

Reply:
Yes.